# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: \n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-08-04 18:50+0000\n"
"PO-Revision-Date: 2017-03-11 12:10+0100\n"
"Last-Translator: \n"
"Language-Team: \n"
"Language: pl\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n==1 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 "
"|| n%100>=20) ? 1 : 2);\n"
"X-Generator: Poedit 1.8.11\n"

#: env/lib/python3.7/site-packages/rest_auth/registration/serializers.py:67
msgid "View is not defined, pass it as a context variable"
msgstr "Widok nie został zdefiniowany, przekaż go przez zmienną \"context\""

#: env/lib/python3.7/site-packages/rest_auth/registration/serializers.py:72
msgid "Define adapter_class in view"
msgstr "Zdefiniuj \"adapter_class\" w widoku"

#: env/lib/python3.7/site-packages/rest_auth/registration/serializers.py:91
msgid "Define callback_url in view"
msgstr "Zdefiniuj \"callback_url\" w widoku"

#: env/lib/python3.7/site-packages/rest_auth/registration/serializers.py:95
msgid "Define client_class in view"
msgstr "Zdefiniuj \"client_class\" w widoku"

#: env/lib/python3.7/site-packages/rest_auth/registration/serializers.py:116
msgid "Incorrect input. access_token or code is required."
msgstr "Podano błędne dane. \"access_token\" lub \"code\" są wymagne."

#: env/lib/python3.7/site-packages/rest_auth/registration/serializers.py:125
msgid "Incorrect value"
msgstr "Niepoprawna wartość."

#: env/lib/python3.7/site-packages/rest_auth/registration/serializers.py:139
#, fuzzy
#| msgid "A user is already registered with this e-mail address."
msgid "User is already registered with this e-mail address."
msgstr "Istnieje już użytkownik z takim adresem email."

#: env/lib/python3.7/site-packages/rest_auth/registration/serializers.py:185
msgid "A user is already registered with this e-mail address."
msgstr "Istnieje już użytkownik z takim adresem email."

#: env/lib/python3.7/site-packages/rest_auth/registration/serializers.py:193
msgid "The two password fields didn't match."
msgstr "Hasła nie są identyczne."

#: env/lib/python3.7/site-packages/rest_auth/registration/views.py:51
msgid "Verification e-mail sent."
msgstr "Email weryfikacyjny został wysłany."

#: env/lib/python3.7/site-packages/rest_auth/registration/views.py:98
msgid "ok"
msgstr "ok"

#: env/lib/python3.7/site-packages/rest_auth/serializers.py:33
msgid "Must include \"email\" and \"password\"."
msgstr "Musisz podać email i hasło."

#: env/lib/python3.7/site-packages/rest_auth/serializers.py:44
msgid "Must include \"username\" and \"password\"."
msgstr "Musisz podać nazwę użytkownika i hasło."

#: env/lib/python3.7/site-packages/rest_auth/serializers.py:57
msgid "Must include either \"username\" or \"email\" and \"password\"."
msgstr "Musisz podać nazwę użytkownika (lub email) i hasło."

#: env/lib/python3.7/site-packages/rest_auth/serializers.py:98
msgid "User account is disabled."
msgstr "Konto użytkownika zostało wyłączone."

#: env/lib/python3.7/site-packages/rest_auth/serializers.py:101
msgid "Unable to log in with provided credentials."
msgstr "Podane dane do logowania są niepoprawne."

#: env/lib/python3.7/site-packages/rest_auth/serializers.py:110
msgid "E-mail is not verified."
msgstr "Email nie został zweryfikowany."

#: env/lib/python3.7/site-packages/rest_auth/views.py:137
msgid "Successfully logged out."
msgstr "Wylogowano."

#: env/lib/python3.7/site-packages/rest_auth/views.py:190
msgid "Password reset e-mail has been sent."
msgstr "Email z linkiem do resetu hasła został wysłany."

#: env/lib/python3.7/site-packages/rest_auth/views.py:216
msgid "Password has been reset with the new password."
msgstr "Hasło zostało zresetowane."

#: env/lib/python3.7/site-packages/rest_auth/views.py:238
msgid "New password has been saved."
msgstr "Nowe hasło zostało zapisane."
